packages/lib-react-components/test/setup.js

```diff
@@ -37,6 +37,7 @@ class ResizeObserver {
 
 window.ResizeObserver = ResizeObserver
 window.scrollTo = () => true
+window.cancelAnimationFrame = () => true
```

Comment:
What component test is this required for?

Reply:
One of the `@visx` libraries, but I can't remember which one. We could figure it out by commenting this out and seeing what breaks.

Reply:
I think this might have been for `@visx/axis`, which I took out of this PR after removing axes from the preview charts. TippyJS uses `cancelAnimationFrame` too.